pkg/proxy/var.go

```diff
@@ -50,6 +50,7 @@ var (
 		variable.NewStringVariable(types.VarUpstreamHost, nil, upstreamHostGetter, nil, 0),
 		variable.NewStringVariable(types.VarUpstreamTransportFailureReason, nil, upstreamTransportFailureReasonGetter, nil, 0),
 		variable.NewStringVariable(types.VarUpstreamCluster, nil, upstreamClusterGetter, nil, 0),
+		variable.NewStringVariable(types.VarUpstreamConnectID, nil, upstreamConnectId, nil, 0),
```

Comment:
we should use `variable.DefaultStringSetter` to set variable, you can refer to `types.VarProxyGlobalTimeout`